modules/onboard-azure-eventhub/variables.tf

```diff
@@ -0,0 +1,175 @@
+  description = "Unique identifier (UID) attached to the jSonar machine controlling the asset"
+  type        = string
+}
+
+variable "azure_eventhub_parent_asset_id" {
+  description = "The asset_id of the azure asset that is sending its audit logs to this AZURE EVENTHUB asset."
+  type        = string
+  default     = null
+}
+
+variable "azure_eventhub_region" {
+  description = "Azure region containing the Event Hub."
+  type        = string
+  default     = null
+}
+
+variable "eventhub_message_retention" {
+  description = "Specifies the number of days to retain the events for this Event Hub."
+  type        = number
+  default     = 1
+}
+
+variable "eventhub_name" {
+  description = "Specifies the name of the Event Hub resource. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "eventhub_partition_count" {
+  description = "Specifies the current number of shards on the Event Hub. Note: partition_count cannot be changed unless Eventhub Namespace SKU is Premium and cannot be decreased."
+  type        = number
+  default     = 1
+}
+
+variable "eventhub_resource_group_name" {
+  description = "The name of the resource group in which the Event Hub's parent Namespace exists. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "eventhub_status" {
+  description = "Specifies the status of the Event Hub resource. Possible values are Active, Disabled and SendDisabled. Defaults to Active."
+  type        = string
+  default     = "Active"
+  validation {
+    condition     = contains(["Active", "Disabled", "SendDisabled"], var.eventhub_status)
+    error_message = "Invalid sku. Possible values are Active, Disabled and SendDisabled."
+  }
+}
+
+variable "eventhub_namespace_capacity" {
+  description = "Specifies the Capacity / Throughput Units for a Standard SKU namespace. Defaults to 1."
+  type        = number
+  default     = 1
+}
+
+variable "eventhub_namespace_location" {
+  description = "Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "eventhub_namespace_name" {
+  description = "Specifies the name of the Event Hub Namespace resource. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "eventhub_namespace_public_network_access_enabled" {
+  description = "Is public network access enabled for the Event Hub Namespace? Defaults to true."
+  type        = bool
+  default     = true
+}
+
+variable "eventhub_namespace_resource_group_name" {
+  description = "The name of the resource group in which to create the namespace. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "eventhub_namespace_sku" {
+  description = "Defines which tier to use. Valid options are Basic, Standard, and Premium. Please note that setting this field to Premium will force the creation of a new resource."
```

Comment:
Do we want to name the default value in the description so that it appears in the docs?